README.md

```diff
@@ -40,7 +40,7 @@
 Only supports the <a href="https://github.com/LibreTranslate/LibreTranslate">LibreTranslate API</a> and therefore is fully libre.
 
 ### Translate You
-[**NonFreeNet anti-feature**](https://f-droid.org/docs/Anti-Features/#NonFreeNet)
+[**NonFreeNet anti-feature**](https://f-droid.org/docs/Anti-Features/#NonFreeNet): **Due to supports translation engines that are not open source.**
```

Comment:
My English sucks but I think we have to use `supporting` instead of `supports` here :)